.eslintignore

```diff
@@ -1,5 +1,8 @@
-coverage
-dist
-docs
-tmp
-blueprints/*/files
+!.eslintrc.js
+!.template-lintrc.js
+/coverage
+/dist
+/docs
+/tmp
+/vendor
+/blueprints/*/files
```

Comment:
FYI: eslint itself uses /** in its own repo: https://github.com/eslint/eslint/blob/master/.eslintignore